docs/components/modeler/web-modeler/launch-web-modeler.md

```diff
@@ -1,29 +1,21 @@
 ---
 id: launch-web-modeler
 title: Launch Web Modeler
-description: "Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately. To launch Web Modeler, take these steps."
+description: "Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms."
 ---
 
 <span class="badge badge--cloud">Camunda 8 only</span>
 
-Since Web Modeler is embedded in the Camunda 8 Console, it does not need to be started separately.
+Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms.
```

Comment:
```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda, where you can also create BPMN diagrams, DMN diagrams, or forms.
```
My brain wants an additional comma here. WDYT? If so, please also add it to the metadata description.

Reply:
I get this sense also - sometimes when I can't decide which version is easier to read, I'll just make two sentences for clarity?
```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda. Web Modeler also allows you to create BPMN diagrams, DMN diagrams, and forms.
```

Reply:
Updated -- thank you both!